repos.bzl

```diff
@@ -15,6 +15,39 @@
 load("@bazel_gazelle//:deps.bzl", "go_repository")
 
 def go_repositories():
+    go_repository(
```

Comment:
I hope this file was generated

Reply:
Yes, this is what `update-deps` generates (via gazelle)